types/intercom-web/index.d.ts

```diff
@@ -0,0 +1,57 @@
+// Type definitions for Intercom Web API 2.8
+// Project: https://docs.intercom.io/
+//            configure-intercom-for-your-product-or-site/
+//            customize-the-intercom-messenger/the-intercom-javascript-api
+// Definitions by: Andrew Fong <https://github.com/fongandrew>
+//                 Samer Albahra <https://github.com/salbahra>
+// Definitions: https://github.com/typed-contrib/knockout
```

Comment:
?

Reply:
Thank you for catching this. As mentioned this was originally written by @fongandrew.

Reply:
Whoops. Copy-paste error. That should definitely be fixed.